app/models/npq_course.rb

```diff
@@ -7,16 +7,16 @@ def self.identifiers
     pluck(:identifier)
   end
 
-  def self.schedule_for(npq_course)
+  def self.schedule_for(npq_course:, cohort: Cohort.find_by!(start_year: 2021))
```

Comment:
i have left this as class method. i'm not sure why this wasn't implemented an instance method on the course in the first place as opposed to a class method that takes an instance of itself

Reply:
historically this was on the Schedule class because it was doing lots of lookup on the constants inside the Schedule namespace, then it got moved to this class. It would indeed be a slightly nicer